### DISCUSSION TITLE
posts/3rd_term/study/2023-11-06-book-study_1st/

### DISCUSSION THREAD

**Discussion by giscus** [gdsc-seoultech/blog-comments#274] [Announcements]
# posts/3rd_term/study/2023-11-06-book-study_1st/

북스터디 1,2 주차

https://gdsc-seoultech.github.io/posts/3rd_term/study/2023-11-06-book-study_1st/

**Comment by Hong-Mu** [comment]
저 백엔드한다고 말해서 웹코어로 전직시켜준건가요? 감사합니다 🙌

**Reply by Changha-dev** [reply]
ㅋㅋㅋㅋㅋㅋ 바로 수정해버리기
